src/python/library/tritonclient/grpc/_client.py

```diff
@@ -365,10 +369,7 @@ def get_server_metadata(self, headers=None, as_json=False):
             If unable to get server metadata.
 
         """
-        if headers is not None:
-            metadata = headers.items()
-        else:
-            metadata = ()
+        metadata = self._pre_api_call(headers)
```

Comment:
metadata seems a little overloaded here as well as request. Should this be request_metadata and request_headers?

Reply:
Fixed.